client/components/cart-summary-items.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+export default class CartSummaryItems extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.fixPrice = this.fixPrice.bind(this);
+  }
+
+  fixPrice(price) {
+    const fixedPrice = price.toFixed();
+    if (fixedPrice.length === 3) {
+      return '$' + fixedPrice[0] + '.' + fixedPrice[1] + fixedPrice[2];
+    } else if (fixedPrice.length === 4) {
+      return '$' + fixedPrice[0] + fixedPrice[1] + '.' + fixedPrice[2] + fixedPrice[3];
+    }
```

Comment:
If you divide the prices by 100, you don't have to care how many digits are in it. You can pass `2` to `.toFixed()` to put the decimal in the right spot.